packages/xchain-thorchain-amm/__e2e__/thornode-checkTx.ts

```diff
@@ -0,0 +1,14 @@
+import { Network } from '@xchainjs/xchain-client'
```

Comment:
this file name needs to end in *e2e.ts in order to be picked up by 'yarn e2e'

Reply:
oh yes of course